model/project_parser.go

```diff
@@ -737,14 +738,16 @@ func LoadProjectInto(ctx context.Context, data []byte, opts *GetProjectOpts, ide
 		}
 	}
 
-	intermediateProject.Include = nil
-
-	// return project even with errors
+	// Return project even with errors.
 	p, err := TranslateProject(intermediateProject)
 	if p != nil {
 		*project = *p
 	}
 	project.Identifier = identifier
+
+	// Don't remove includes until project is translated.
```

Comment:
nit: should this instead say "Remove includes once the project is translated", since it's doing the removal right here?

Can we also clarify why we need to remove them?

Reply:
It looks like the parser project that's returned is still used [to save the project yaml](https://github.com/evergreen-ci/evergreen/blob/DEVPROD-731/model/patch_lifecycle.go#L313) and I don't want to change any expected behavior 

But it makes much more sense to update the comment

Reply:
Thanks! Can we also include a mention of what removing the includes is intended to do? Like, why not just leave it in as-is?